server/createNewDeal.ts

```diff
@@ -0,0 +1,21 @@
+"use server";
+
+import prisma from "@/lib/prisma";
+
+export async function createNewDeal(
+  item: string,
+  pricePerUnit: number,
+  unitType: string,
+  dateObserved: Date,
+  storeID: string
+) {
+  await prisma.deal.create({
+    data: {
+      item: item,
+      pricePerUnit: pricePerUnit,
+      unitType: unitType,
+      dateObserved: dateObserved,
+      storeID: storeID,
```

Comment:
when the object key and variable name happen to be exactly the same, you can use a shortcut syntax to assign them like
```
data: {
      ite:,
      pricePerUnit,
      ...etc
}
```

Reply:
https://github.com/amiaoh/groceryDatabaseApp/pull/30/commits/9663a8ffd187cf6ddc9e8e7ff6ff1a12b9ab1e97